packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -140,25 +183,42 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> {
   public componentDidMount(): void {
     // If we have already accessed the width and height
     // We do not wan't to do it again.
+    const entityInnerHostRect = this._entityInnerHostElement.offsetWidth;
+
     if (this.state.isMeasuring) {
       this._async.setTimeout((): void => {
+
         if ((this.state.entityInnerHostRect.width + this.state.entityInnerHostRect.width) === 0) {
+
+          // @TODO Eventually we need to add the various directions
```

Comment:
With some tweaks we could make the current beak positioning method public from positioning.